docs/doc/15-sql-functions/04-array-functions/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: 'Array Functions'
+---
+
+Array Functions
+
+## Syntax
+
+| Function    | Description             |  Example | Result
+| ----------- | ------------------ |  ---------- | --------|
+| `get(array1, index)`  | Get item from an array by index     |  `get([1,2], 2)`         |  2
+| `length(array1)`  | Output the length of an array    |  `length([1,2])`         |  2
+| `array_concat(array1, array2)`  | Concat two arrays     |  `array_concat([1,2], [3, 4]`         |  [1,2,3,4]
+| `array_contains(array, item)`  | Check if array has some item     |  `array_contains([1,2], 1)`         |  1
+| `array_indexof(array, item)`  | Returns the index of the element if the list contains the element.	list_contains([1, 2, NULL],      |  `array_indexof([1,2,9], 9);`         |  3
+| `array_slice(array, start, end)`  | Extract a sublist using slice conventions     |  `array_slice([1,21,32,4], 2, 3)`         |  `[21, 32]`
+| `array_sort(array)`  | Sort an array     |  `array_sort([1,4,3,2])`         |  `[1,2,3,4]`
```

Comment:
`array_sort` function have two additional optional arguments.
The second optional argument is the sort order, `ASC` and `DESC` are supported, the default value is `ASC`.
The third optional argument is the position of the `NULL` value, `NULLS FIRST` and `NULLS LAST` are supported, the default is `NULLS FIRST`.